src/main/java/org/osm2world/console/OSM2World.java

```diff
@@ -238,6 +238,13 @@ public static Configuration loadConfigFiles(@Nullable LevelOfDetail lod, File...
 			config.load(it);
 		}
 
+		Arrays.stream(configFiles)
+			.filter(f -> f.exists())
+			.findFirst()
+			.ifPresent(f -> {
+				config.addProperty("configPath", f.getParent());
```

Comment:
Consider using `config.setFile` to store the path instead of a custom property.

Reply:
I think during load and parsing it's done automatically. So one option would be to rely on `config.getBasePath` and remove this changes. But that would mean that I'd have to typecheck config in `ConfigUtil.resolveFileConfigProperty` to check that `config.getBasePath` exists. I'm fine with typecheck itself, but i don't like that I would have to rely on config being of a certain type. What if you'll change it to some other descendant of `Configuration` in the future.